### PR TITLE
Update Parachain Related Content

### PR DESCRIPTION
**Pull request by salmad3**
Part of #2692

### CODE REVIEW

File: docs/learn/learn-bridges.md [hunk 1]
```diff
@@ -8,49 +8,53 @@ slug: ../learn-bridges
...
+{{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }}
 (non-Substrate), it will be necessary to deploy bridge contracts (see below).
 
 ### via Smart Contracts
 
 Given the generality of blockchain platforms with Turing-complete smart contract languages, it is
-possible to bridge Polkadot and any other smart contract capable blockchain (Ethereum, EOS, Tezos,
-etc).
+possible to bridge {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} and any other 
+smart contract capable blockchain (Ethereum, EOS, Tezos, etc).
```
Comment (emresurmeli) [inline docs/learn/learn-bridges.md]: Personal reservation, I'd refrain from mentioning EOS on the wiki... based on their repos on github it doesn't seem to be maintained any longer, with the last commit on the C++ implementation being on Jul 20th (https://github.com/EOSIO/eos)

File: docs/learn/learn-common-good-chains.md [hunk 2]
```diff
@@ -64,13 +64,15 @@ likely be approved by the network stakeholders only in rare scenarios.
...
 [Statemint](https://github.com/paritytech/statemint) (and its cousin *Statemine* on Kusama) will
-likely be one of the first common good parachains. Statemint is a public utility chain in that it
-adds functionality not available in the Relay Chain, namely the creation and management of assets.
+likely be one of the first common good parachains.
 
-Statemint will support both fungible and non-fungible assets. The chain offers an interface similar
-to ERC-20 for fungible assets and ERC-721 for non-fungible tokens. These interfaces are in the logic
-of the chain itself; by encoding this logic directly into the Statemint runtime, token storage and
-actions do not need to be metered and can happen faster and cheaper.
+> Statemine is the first common good parachain.
```
Comment (emresurmeli) [inline docs/learn/learn-common-good-chains.md:69]: We can link to Joes addition of /learn-assets

File: docs/learn/learn-crowdloans.md [hunk 3]
```diff
@@ -25,36 +25,40 @@ campaign just because they do not secure a slot on their first attempt.
 When setting the parameters of a crowdloan campaign, consider the following:
```
Comment (emresurmeli) [inline docs/learn/learn-crowdloans.md]: I would mention that crowdloans can begin well before an auction slot is opened.

File: docs/learn/learn-common-good-chains.md (hunk 2 above)
Comment (salmad3) [reply docs/learn/learn-common-good-chains.md:69]: Will do so once learn-assets is merged

File: docs/learn/learn-bridges.md (hunk 1 above)
Comment (salmad3) [reply docs/learn/learn-bridges.md]: It seems there is maintenance activity at the very least. We can keep it "generic" and word it like `smart contract capable blockchain`.

**Pull request merged**